File: Practices1/ProjectEuler/p10.c
#include<stdio.h>
#include<stdbool.h>
bool isPrime(int num) {
    if(num > 1) {
        for(int i = 2; i < num; i++) {
            if(num % i == 0)
                return false;
        }
        return true;
    }
    else return false;
}
int main(void) {
    int total = 0;
    for(int i=0; i < 2000000; i++) {
        if(isPrime(i) == true)
            total += i;
    }
    printf("%d\n", total);
}

File: Practices1/ProjectEuler/p10.py
# Project Euler Problem Set 10

def isPrime(num):
    if num > 1:
        for n in range(2, num):
            if num % n == 0:
                return False
        else:
            return True
    else:
        return False

if __name__=="__main__":
    total = 0
    for i in range(2_000_000):
        if isPrime(i):
            total += i
    print(total)